syft/frameworks/torch/tensors/interpreters/gradients.py

```diff
@@ -213,3 +213,14 @@ def gradient(self, grad):
         zero = self.self_ * 0
         gt_zero = self.self_ > zero
         return (gt_zero * grad,)
+
+
+class ViewBackward(GradFunc):
+    def __init__(self, self_):
+        super().__init__(self, self_)
```

Comment:
Try adding *args, **kwargs into arguments list, since `.view(...)` args are forwarded to grad_fn's init in autograd.py#L165:
`result.grad_fn = grad_fn(self, *args, **kwargs)`

Reply:
Thanks for feedback Vova. Great learnings for me.